.storybook/themes/news.scss

```diff
@@ -1367,6 +1367,40 @@
 					margin: 0,
 					padding: 0,
 				),
+				medium-promo: (
+					components: (
+						attribution: (
+							display: block,
+							margin-top: var(--global-spacing-4),
+						),
+						media-item: (
+							float: right,
+							max-width: 100px,
+						),
+						heading: (
+							font-size: var(--global-font-size-7),
+							font-weight: var(--global-font-weight-7),
+							line-height: var(--global-line-height-4),
```

Comment:
```suggestion
							font-size: var(--heading-level-5-font-size),
							font-weight: var(--heading-level-5-font-weight),
							line-height: var(--heading-level-5-line-height),
```

Reply:
Thanks! Changes have been made! Committed before I saw this commit suggestion.